frontend/app/templates/index.hbs

```diff
@@ -109,6 +109,16 @@
                                 </div>
                             </section>
                             <input type="hidden" name="custfont" value="">
+                            <li>
+                            <li>
+                                <input type="radio" name="backsource" id="text-color"> Choose Text Color</li>
```

Comment:
Can we have a system in which we can change the text style and text colour both simultaneously. I will be happy if you could make two different sections.

Reply:
Just remove this radio